INN/doubleclick-for-wp
# Gutenberg Support

🚨 this is a WIP 🚨 

## Changes

- adds gutenblock
    - [x] identifier
    - [x] lazyload 
    - [x] sizes
    - [x] customClassName
    - [x] align
    - [x] alignwide
- slight changes to PR template to save keystrokes.

![screen shot 2018-11-01 at 8 15 23 pm](https://user-images.githubusercontent.com/1754187/47886857-e5253a80-de12-11e8-8131-aad29012e6a9.png)

## Why

For #70 

## Testing/Questions

Questions that need to be answered before merging:

- [x] does it work in 4.9.8?
- [x] does it work in 5.0-beta2?
- [x] do widgets still work?
- [x] does the widget accept the alignment properties?

Steps to test this PR:

1. Install plugin
2. Configure the plugin's breakpoints.
3. In a post, add a block!
4. Configure the block's sizes at its breakpoints
5. Configure the block's ad unit code
6. Set an alignment and check that that's respected.
